uw-frame-components/portal/notifications/directives.js

```diff
@@ -11,16 +11,17 @@ define(['angular', 'require'], function(angular, require){
         }
     });
 
-    app.directive('notificationBell', function(){
+    app.directive('notificationPartials', function(){
```

Comment:
At the risk of being pedantic: "partials" is an AngularJS implementation detail, it's not awesome to expose it in the name of the resulting directive HTML element. What if we were to go a different direction here, maybe invent an AngularJS-portal namespace, and call the directive "ap-notification", for a resulting semantic-ly-named HTML element (directive) usage looking like `<ap-notification mode="priority" class="visible-xs"></ap-notification>` ?

This seems nicely readable. What is it? An AngularJS-portal notification. Oh. What mode is it in? Priority.